cloud/cloud.py

```diff
@@ -60,6 +61,9 @@ def make_cloud_config(config, position):
         for k, v in getattr(config, position + '_mysql').iteritems():
             cloud_config['mysql'][k] = v
 
+        for k, v in getattr(config, position + '_rabbit').iteritems():
+            cloud_config['rabbit'][k] = v
+
```

Comment:
OMG :) Is there any way we stop adding these? :) All of them are *completely* identical other than the attribute name. Probably OK for now, but needs refactoring in future.